driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -64,6 +64,7 @@
 
   private static final boolean HOST_METRICS_ENABLED =
       Boolean.getBoolean("com.datastax.driver.HOST_METRICS_ENABLED");
+  static final String LOG_REQUEST_WARNINGS_PROPERTY = "advanced.request.log-warnings";
```

Comment:
In driver 3 we've been following a different naming convention for this kind of system property, see the previous line for example.
Also, as much as I dislike the idea of logging those warnings, I think we should do it by default, because the assumption is that users will completely miss them otherwise.

So I would invert the property and call it `com.datastax.driver.DISABLE_QUERY_WARNING_LOGS`.